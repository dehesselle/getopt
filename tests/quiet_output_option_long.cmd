#!/bin/sh
"$1" --quiet-output -o a,b: -l long1,long2 -- -c --unknown --long -b
