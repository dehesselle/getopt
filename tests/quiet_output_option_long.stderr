$0: invalid option -- 'c'
$0: unrecognized option '--unknown'
$0: option '--long' is ambiguous
$0: option requires an argument -- 'b'
